src/onnx/parse_resize.cpp

```diff
@@ -138,8 +135,7 @@ calc_neighbor_points(const std::vector<std::vector<std::vector<std::size_t>>>& v
                            return dim;
                        });
     }
-
-    return calc_neighbor_points(vvv_ind, i_dim + 1, vec_dims1, in_s);
+    return calc_neighbor_points(vvv_ind, i_dim + 1, std::move(vec_dims1), in_s);
```

Comment:
You should do `vec_dims.clear()` before returning.

Reply:
duh. yes, the original purpose..; thanks. :-)

Reply:
(Fixed already. Thanks.)